lgtm/__init__.py

```diff
@@ -3,7 +3,7 @@
 import integrations
 
 
-def pull_request_ready_to_merge(github_token, org, repo, pr_number, owners_file='OWNERS'):
+def pull_request_ready_to_merge(github_token, org, repo, pr_number, owners_file='OWNERS', options={}):
```

Comment:
Interesting. I don't think I've ever run into this problem, because mostly I probably don't use arrays for defaults, and I don't think this particular gotcha is applicable to hashes. I'll change it to your suggestion.